### DISCUSSION TITLE
Can't see the configuration for schedule power output from the solarbanks

### DISCUSSION THREAD

**Discussion by longliveenduro** [thomluther/ha-anker-solix#78] [Q&A] [answered]
Hi @thomluther, thank you very much for your work.
It looks fine so far.
I added my 3 solarbanks today. I am struggling to find the configuration for the "Output to family" schedule/times though?

Any hints where I can find that/ how I activate it?

Thank you very much.

**Comment by thomluther** [comment]
Are you talking about solarbank 1 or 2?
Solarbank 2 is not supported, solarbank 1 should work with 1.3.1. This now creates all schedule entities even if no schedule is defined yet.

For Solarbank 2 read this first:
[Help required to add support for Solarbank 2](https://github.com/thomluther/anker-solix-api/discussions/113)

**Comment by longliveenduro** [comment]
@thomluther   Thanks Thomas for the fast answer. 

I have 3 Solarbank 1 (First generation). I am new to HomeAssistant, so I maybe missed the display and configuration of the schedules.

I used my normal user / owner and just stopped the app on my phone and never started it again since then. (I want to use Home Assistant and your integration solely in the future).

On my Overview in Home Assistant (I still use HA managed Overview) I see all my solarbanks and all 3 systems for it. (I used separate systems for each solarbank). I also can change the values, so everything seems to work. I just have no idea where to look for the schedule display and setting them. They were not automatically added by HA to my Overview tab. Is there sth I need to configure?

I am a developer, so you can give me all the details, if you want.

Cheers Chris (from Augsburg, Bavaria)

(![Screenshot_20240630_113452_Home Assistant](https://github.com/thomluther/hacs-anker-solix/assets/599308/e410c906-09c7-4af9-bf07-39ecca92fa7d))

**Comment by thomluther** [comment]
Hi @longliveenduro 
Have you carefully read the [INFO](https://github.com/thomluther/hacs-anker-solix/blob/main/INFO.md)?
I think all you need is described there

**Reply by longliveenduro** [reply]
Hi @thomluther, thanks, got it sorted out. Thanks for the hint to dig in.

Only thing I need to do now is to setup the script for changing the schedules.
Not easy to understand on my first read, but I will not give up and try the next days.

Thank you so much for your hard work!

**Comment by longliveenduro** [comment, marked as answer]
Carefully read INFO
